### DISCUSSION TITLE
P-dropdown is not properly working

### DISCUSSION THREAD

**Discussion by anantkj** [primefaces/community#380] [PrimeNG] [answered]
P-dropdown is not properly working  with recent version update , it seems changes in dropdown functionality because it was working before 
showClear attribute is not working filter is also not working

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We've fixed this issue in https://github.com/primefaces/primeng/issues/13996 and the fix will be available with upcoming release 16.8.1 this week.

**Reply by ThoSap** [reply]
PrimeNG 16.7.1 is out now
https://github.com/primefaces/primeng/blob/master/CHANGELOG.md#16.7.1
